core/examples/json/ProcessDescription.json

```diff
@@ -151,21 +150,24 @@
     "imagesInput": {
       "title": "Inline Images Value Input",
       "description": "This is an example of an image input.  In this case, the input is an array of up to 150 images that might, for example, be a set of tiles.  The oneOf[] conditional is used to indicate the acceptable image content types; GeoTIFF and JPEG 2000 in this case.  Each input image in the input array can be included inline in the execute request as a base64-encoded string or referenced using the link.yaml schema.  The use of a base64-encoded string is implied by the specification and does not need to be specified in the definition of the input.",
-      "minOccurs": 1,
-      "maxOccurs": 150,
       "schema": {
-        "oneOf": [
-          {
-            "type": "string",
-            "contentEncoding": "binary",
-            "contentMediaType": "image/tiff; application=geotiff"
-          },
-          {
-            "type": "string",
-            "contentEncoding": "binary",
-            "contentMediaType": "image/jp2"
-          }
-        ]
+        "type": "array",
+	"minItems": 1,
+	"maxItems": 150,
+        "items": {
```

Comment:
Missing the single item (non-array) representation.

Reply:
I'm updating this PR to undo the deprecation of minOccurs and maxOccurs.

Reply:
The `imagesInput` example now uses minOccurs/maxOccurs.